src/gui/gui.cc

```diff
@@ -325,7 +325,7 @@ void PCSX::GUI::endFrame() {
     bool changed = false;
 
     if (m_fullscreenRender) {
-        ImTextureID texture = reinterpret_cast<ImTextureID*>(&m_offscreenTextures[m_currentTexture]);
+        ImTextureID texture = reinterpret_cast<ImTextureID*>(m_offscreenTextures[m_currentTexture]);
```

Comment:
Woops ;-)

Reply:
Do you mean I need to revert this?

Reply:
No, no, it was a problem from the original change you've done. That's the proper fix here.

Reply:
Yes, I haven't seen it then. Here it is some docs if you need: https://github.com/ocornut/imgui/issues/1840